### ISSUE TITLE
Icon not shown in desktop notifier

### ISSUE THREAD

**Issue by Echecivuole** [nunomaduro/laravel-desktop-notifier#28]
Hi, the desktop notifier works fine, but even adding the code
`resource_path('image.png')`
the image is not showing. I read some other posts for which it would need required to install other package into the OSX o Windows pc, but this cannot be asked to all users using this feature. 

Any solution?

**Comment by owenvoke** [comment]
Not really, under the hood this package uses [JoliNotif](https://github.com/jolicode/JoliNotif), so unless there was support for it in that, I don't think we can really do this.

**Comment by owenvoke** [comment]
It looks like there is only support for it in [the `TerminalNotifierNotifier` class](https://github.com/jolicode/JoliNotif/blob/master/doc/02-notification.md#url), however, you'd have to use the underlying notification class directly and append `$notification->addOption('url', 'https://example.com');` such as (this untested example):

```php
app(\Joli\JoliNotif\Notifier\TerminalNotifierNotifier::class)->send(
    app(\Joli\JoliNotif\Notification::class)
      ->setTitle($text)
      ->setBody($body)
      ->setOption('url', 'https://example.com');
);
```

Note: This also would only work on Mac OS X 10.8 and higher, and requires [`terminal-notifier`](https://github.com/julienXX/terminal-notifier) to be installed. 😕

**Comment by JonPurvis** [comment]
Hey @owenvoke 👋 

Is there any chance of this being looked into further? If not, it might be worth updating [this documentation](https://laravel-zero.com/docs/send-desktop-notifications). It would be a shame to not have this part work 😢

**Comment by igorsgm** [comment]
Yeah, I'm also facing the same issue on macOS Ventura

**Comment by owenvoke** [comment]
Please do feel free to open a PR to JoliNotif to add native support, or to the docs mentioning that this is not possible.
